src/Dmishh/Bundle/SettingsBundle/Controller/SettingsController.php

```diff
@@ -107,4 +110,20 @@ protected function manage(Request $request, SettingsOwnerInterface $owner = null
             )
         );
     }
+
+    /**
+     * Get SecurityContext service
+     * @param string $service The service name
+     *
+     * @return mixed The service
+     */
+    private function getSecurityContext($service)
```

Comment:
A bit weird signature for this method. I would remove it and implement all the logic for getting `SecurityContext` inside the method without passing any params.

Something like:
```
if ($this->has('security.token_storage')) {
    return $this->get('security.token_storage');
} else {
    return $this->get('security.context')
}
```